ui-tests/src/test/java/org/openmrs/reference/page/ManageHtmlFormsPage.java

```diff
@@ -0,0 +1,24 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class ManageHtmlFormsPage extends Page {
+    private static final By NEW_HTML_FORM = By.cssSelector("a[href='htmlForm.form']");
```

Comment:
Refer to the artifacts collected [here](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/476#issuecomment-870471982) to ensure quality assurance to align with software standards